ignite-base/App/Services/RehydrationServices.js

```diff
@@ -18,14 +19,14 @@ const updateReducers = (store) => {
         preview: 'Reducer Version Change Detected',
         important: true
       })
-      // Purge store
-      persistStore(store, config).purge()
+      // Purge store and dispatch startup
+       persistStore(store, config,()=> {store.dispatch(StartupActions.startup());}).purge()
       AsyncStorage.setItem('reducerVersion', reducerVersion)
     } else {
-      persistStore(store, config)
+      persistStore(store, config,()=> {store.dispatch(StartupActions.startup());});
```

Comment:
Actually, you can create a `const startup = () => store.dispatch(StartupActions.startup())`

And then swap out all the anonymous arrow functions for `startup`, which reads reallllly nice.